vllm/v1/engine/async_llm.py

```diff
@@ -146,6 +170,7 @@ async def add_request(
         detokenizer_req, engine_core_req = self.processor.process_inputs(
             request_id, prompt, params, arrival_time, lora_request,
             trace_headers, prompt_adapter_request, priority)
+        self.stats_manager.record_engine_input(engine_core_req)
```

Comment:
do we need this for `LLM` as well?

Reply:
Also, where is `arrival_time` used?

Reply:
oh yes - the prototype was only on async llm, but it should be the same for sync llm engine.